lib/custom.rb

```diff
@@ -0,0 +1,24 @@
+require "response"
+
+class Custom
+  def initialize(env)
+    @env = env
+  end
+
+  def index
+    Response.new.tap do |response|
```

Comment:
I'm not sure about using #tap. I avoid it as much as possible, usually it just obfuscates what's actually going on. Since Response is your class, why not add constructor arguments: Response.new("Hello World", 200)